### PR TITLE
shell scripts: fix them by following shellcheck recommendations

### PR DESCRIPTION
**Pull request by xavier-rbz**

### CODE REVIEW

File: deploy/curiefense-helm/deploy.sh [hunk 1]
```diff
@@ -29,11 +29,10 @@ if ! kubectl get namespaces|grep -q curiefense; then
     echo "curiefense namespace created"
 fi
 
-helm upgrade --install --namespace curiefense --reuse-values $HELM_ARGS \
+# shellcheck disable=SC2086
+if helm upgrade --install --namespace curiefense --reuse-values ${HELM_ARGS} \
```
Comment (flaper87) [inline deploy/curiefense-helm/deploy.sh:33]:
```suggestion
if ! helm upgrade --install --namespace curiefense --reuse-values ${HELM_ARGS} \
```

We only want to describe pods/fetch logs if the deployment fails. Unless I'm missing something, of course.

File: deploy/istio-helm/deploy.sh [hunk 2]
```diff
@@ -29,14 +29,13 @@ if [ -n "$NOPULL" ]; then
     PARAMS+=("--set" "global.imagePullPolicy=Never")
 fi
 
-helm upgrade --install --namespace istio-system --reuse-values \
+# shellcheck disable=SC2086
+if helm upgrade --install --namespace istio-system --reuse-values \
```
Comment (flaper87) [inline deploy/istio-helm/deploy.sh:33]: ditto

**Review by flaper87 — CHANGES_REQUESTED** [review]

File: deploy/curiefense-helm/deploy.sh (hunk 1 above)
Comment (xavier-rbz) [reply deploy/curiefense-helm/deploy.sh]: nice catch. Applied in the next commit

File: deploy/istio-helm/deploy.sh (hunk 2 above)
Comment (xavier-rbz) [reply deploy/istio-helm/deploy.sh]: ditto

**Pull request merged**